backend/ee/danswer/server/query_and_chat/models.py

```diff
@@ -74,4 +74,6 @@ class ChatBasicResponse(BaseModel):
     simple_search_docs: list[SimpleDoc] | None = None
     error_msg: str | None = None
     message_id: int | None = None
```

Comment:
these are the 2 new fields (and one renamed) that we are returning through the simple apis